doc/sai-ptf/config_data/config_t0.md

```diff
@@ -0,0 +1,168 @@
+# Sample T0 Configurations and data  <!-- omit in toc --> 
+- [Overriew](#overriew)
+- [IP and MAC naming convention](#ip-and-mac-naming-convention)
+  - [MAC](#mac)
+  - [IP](#ip)
+- [1. L2 Configurations](#1-l2-configurations)
+  - [1.1 FDB Configuration](#11-fdb-configuration)
+  - [1.2 VLAN configuration](#12-vlan-configuration)
+- [2. L3 configuration](#2-l3-configuration)
+  - [2.1 VLAN Interfaces](#21-vlan-interfaces)
+  - [2.2 LAG configuration](#22-lag-configuration)
+    - [2.2.1 LAG Hash Rule](#221-lag-hash-rule)
+  - [2.3 Route Configuration](#23-route-configuration)
+    - [2.3.1 VLAN interfaces route entries](#231-vlan-interfaces-route-entries)
+    - [2.3.2 LAG Route entry](#232-lag-route-entry)
+  - [2.4 Neighbor Configuration](#24-neighbor-configuration)
+    - [2.4.1 VLAN Neighbors](#241-vlan-neighbors)
+    - [2.4.2 LAG Neighbors](#242-lag-neighbors)
+# Overriew
+This document describes the sample configuration data.
+
+**Note: This configuration focused on T0 topology.**
+
+# IP and MAC naming convention
+In this configuration, we mapped the IP and MAC address into different part of this configuration as below.
+
+## MAC
+For MAC addres, we can use differnt section in MAC address to map different title number.
+The pattern is
+```
+L1_NUM:L2_NUM:L3_NUM:ROLE:EXTRA:SEQ
```

Comment:
@richardyu-ms for mac address, could we follow?
https://howdoesinternetwork.com/2011/mac-address#:~:text=The%20Layer%202%20address%20is%20unique%20on%20the,is%20called%20the%20Media%20Access%20Control%20%28MAC%29%20address.